app/models/manageiq/providers/openshift/container_manager.rb

```diff
@@ -21,9 +28,23 @@ def self.event_monitor_class
     ManageIQ::Providers::Openshift::ContainerManager::EventCatcher
   end
 
+  def self.virtualization_options
+    [
+      {
+        :label => _('Disabled'),
+        :value => 'none',
+      },
+      {
+        :label => _('OpenShift Virtualization'),
+        :value => 'kubevirt',
+        :pivot => 'endpoints.kubevirt.hostname',
+      },
+    ]
+  end
+
   def self.raw_connect(hostname, port, options)
     options[:service] ||= "openshift"
-    send("#{options[:service]}_connect", hostname, port, options)
+    send(:"#{options[:service]}_connect", hostname, port, options)
```

Comment:
Is this necessary?

Reply:
Seems like my rubocop did this automatically haha